commonTypes.h

```diff
@@ -143,6 +143,7 @@ class dfColors
 
 struct GameConfiguration {
     bool overlay_mode;
+    bool autosize_segment;
```

Comment:
don't we need to track autosize for x and y separately since they are configurable separately?